test/package.json

```diff
@@ -28,13 +28,11 @@
     "@bedrock/jsonld-document-loader": "^4.0.0",
     "@bedrock/kms": "^14.0.0",
     "@bedrock/kms-http": "^18.0.0",
-    "@bedrock/ledger-context": "^24.0.0",
     "@bedrock/meter": "^5.0.0",
     "@bedrock/meter-http": "^12.0.0",
     "@bedrock/meter-usage-reporter": "^9.0.0",
     "@bedrock/mongodb": "^10.0.0",
     "@bedrock/multikey-context": "^2.0.0",
-    "@bedrock/oauth2-verifier": "^2.0.0",
```

Comment:
This is a required peer dependency of `@bedrock/service-core` so it should not be removed.

Reply:
I've reverted back the test deps removal here [fd3bbbe](https://github.com/digitalbazaar/bedrock-vc-verifier/pull/55/commits/fd3bbbe89cb8374622ab366d4c0b70df22620185). Not sure why but when I used `npx depcheck` it flagged `@bedrock/service-core` and `@bedrock/ledger-context` as unused packages and removal of those packages didn't cause any issues during installation or when running test.